samples/web/content/apprtc/css/main.css

```diff
@@ -101,28 +101,6 @@ footer > div.active {
   opacity: .8;
 }
 
-@-webkit-keyframes rotate {
```

Comment:
Yes. This was only used for the camera-switch animation, which I've removed. Rotation of the video elements is done via a transform.